roles/control_node/templates/ansible-navigator.yml.j2

```diff
@@ -4,10 +4,10 @@ ansible-navigator:
     inventories:
     - /home/{{username}}/lab_inventory/hosts
 
+  mode: stdout
+  
   execution-environment:
-{% if workshop_type in ['f5'] %}
-    image: quay.io/acme_corp/f5_modules:latest
-{% endif %}
+    image: quay.io/acme_corp/{{ workshop_type }}_ee
```

Comment:
Do we need a conditional here? what if a workshop type doesn't have an EE, example: for RHEL we want to use the default EE so we don't need this line

Reply:
i wonder if we need to have a list somewhere.... like
```
rhel_ee: this
network_ee: that
f5_ee: another_thing
```

so the template just do 
image: "{{workshop_type}}_ee" and does a var lookup?

Reply:
```
rhel_ee: {{ supported_ee }}
network_ee: {{ supported_ee }}
f5_ee: {{ f5_ee }}
security_ee: {{ security_ee }}
```
something like this then